app/models/lead_provider_delivery_partnership.rb

```diff
@@ -1,9 +1,13 @@
 class LeadProviderDeliveryPartnership < ApplicationRecord
+  include DeclarativeTouch
+
   belongs_to :active_lead_provider
   belongs_to :delivery_partner
   has_many :school_partnerships
   has_many :events
 
+  touch -> { delivery_partner }, on_event: %i[create destroy], timestamp_attribute: :api_updated_at
```

Comment:
I couldn't find in ECF1's "touch" model where we touch a delivery partner when a `Partnership` is created or removed. Is it a new behaviour that we're adding in ECF2? If so, it's ok.. 👍

Reply:
My understanding is that we want to touch the `api_updated_at` when any of the serialised response changes, so slightly different from ECF. Is that right @cwrw?